include/nbl/builtin/hlsl/cpp_compat/type_traits.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2022 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+#ifndef _NBL_BUILTIN_HLSL_TYPE_TRAITS_INCLUDED_
+#define _NBL_BUILTIN_HLSL_TYPE_TRAITS_INCLUDED_
+
+// REVIEW-519: We said 1:1 with STL, does that mean without nbl::hlsl namespace so that code can be interoperable with CPP?
```

Comment:
yes `nbl::hlsl` namespace, we're not allowed to add to `std` namespace by the power of C++

Reply:
@kpentaris fix this ASAP before using it

@atkurtul take note